src/base/types/CoreModel.jl

```diff
@@ -96,6 +135,30 @@ Return the stoichiometry of reaction at index `ridx`.
 reaction_stoichiometry(m::CoreModel, ridx)::Dict{String,Float64} =
     Dict(m.mets[k] => v for (k, v) in zip(findnz(m.S[:, ridx])...))
 
+"""
+    grrs(a::CoreModel)::Vector{Maybe{GeneAssociation}}
+
+Get the gene associations in a [`CoreModel`](@ref).
+"""
+grrs(a::CoreModel)::Vector{Maybe{GeneAssociation}} = a.grrs
```

Comment:
Questions:
- This is not used anywhere nor tested?
- If it should be exported, could we use a "full semantics" name such as `reaction_gene_association_vec`?
- Would it make sense to do the same for other models? (ie does the vectorish form of GRRs help somewhere?)

Reply:
(possible usage -- this surely would help the users to dodge the ugly `indexin` in `reaction_gene_association`)

Reply:
I added it for completeness, since there is the same "get" for everything else. And it's not tested, a lot of stuff in `CoreModel` isn't I think.